test/node_smoke_tests/lib/ensure_jquery.js

```diff
@@ -0,0 +1,11 @@
+/* jshint node: true */
+
+"use strict";
+
+// Check if the object we got is the jQuery object by invoking a basic API.
+module.exports = function ensureJQuery( jQuery ) {
+	if ( typeof jQuery.trim !== "function" || jQuery.trim( " ab " ) !== "ab" ) {
```

Comment:
I'd like to see something simpler here for now; maybe `/^jQuery/.test( jQuery.expando )`?